src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.ValidateSignature.cs

```diff
@@ -261,10 +261,7 @@ private static Result<SecurityKey, ExceptionDetail> ValidateSignatureWithKey(
                 else
                     return new ExceptionDetail(
                         new MessageDetail(
-                            TokenLogMessages.IDX10504,
-                            LogHelper.MarkAsSecurityArtifact(
-                                jsonWebToken.EncodedToken,
-                                JwtTokenUtilities.SafeLogJwtToken)),
+                            TokenLogMessages.IDX10500),
```

Comment:
Why did we change this to not return the token?

Reply:
Because 10500 is: "IDX10500: Signature validation failed. No security keys were provided to validate the signature.";

I can keep it as 10504, but I aligned it to return the exact same error that the current path does.

Reply:
But we lost the EncodedToken, please change it back.